src/Cake.Common/Tools/DNVM/Use/DNVMSettings.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.DNVM.Use
+{
+    /// <summary>
+    /// Settings for the dnvm command
+    /// </summary>
+    public class DNVMSettings : ToolSettings
+    {
+        /// <summary>
+        /// Gets or sets the runtime to be used
+        /// </summary>
+        public string Runtime { get; set; }

```

Comment:
The ValidateSet attribute seem to be a powershell item. I'm not sure if its really needed in build.ps1 script as this script know nothing about dnvm, dnu and dnx tools.

Do you have any advice pls ?